docs/features/software-templates/writing-custom-actions.md

```diff
@@ -69,9 +69,9 @@ The `createTemplateAction` takes an object which specifies the following:
 
 You can also choose to define your custom action using JSON schema instead of `zod`:
 
-```ts title="With JSON Schema"
-import { createTemplateAction } from '@backstage/plugin-scaffolder-node';
-import fs from 'fs-extra';
+```ts
+import { createTemplateAction } from '@backstage/plugin-scaffolder-backend';
+import { writeFile } from 'node:fs/promises';
```

Comment:
Not sure we should be documenting `node:`yet. We don't anywhere else and it's just confusing I think at this point.

Reply:
hm, ok, I thought to give some example  how to do such kind of aborting stuff, changed it to `node:fs` as `fs-extra` doesn't support abortable

Reply:
you can just do `fs` instead of `node:fs` :)